addon/extend.js

```diff
@@ -1,4 +1,21 @@
 export { findElement } from './-private/extend/find-element';
 export { findElementWithAssert } from './-private/extend/find-element-with-assert';
 export { buildSelector, getContext, fullScope } from './-private/helpers';
-export { register as registerExecutionContext } from './-private/execution_context';
+import { register as registerExecutionContext } from './-private/execution_context';
+
+import IntegrationNativeContext from './-private/execution_context/integration-native';
+import AcceptanceNativeContext from './-private/execution_context/acceptance-native';
+import IntegrationClassicContext from './-private/execution_context/integration';
+import AcceptanceClassicContext from './-private/execution_context/acceptance';
```

Comment:
> Suggest `IntegrationJQueryContext` and `AcceptanceJQueryContext` to increase clarity. 

Currently `AcceptanceNativeContext ` and `IntegrationNativeContext`  both use `jquery` to find elements.

I'd suggest to rename all the context's module names. 
How about:
- `acceptance-ember-context`, 
- `integration-ember-context` 
- `acceptance-native-dom-context`
- `integration-native-dom-context` 

It seem too verbose to me. But I'm ok with it.

Reply:
How about `acceptance-jquery-event-context`, `acceptance-native-event-context`, etc? And then, `native-event-context` instead for `context.js` above? If the file names are too long, we could add `acceptance` and `integration` folders, each of which could include `native-event-context` and `jquery-event-context`.

Reply:
> How about acceptance-jquery-event-context
- i'm not sure why do you refer to `event`, triggering of events is only a part of context
- `jquery` is not dedicated only for existing contexts. All of contexts(including `native-dom`)  use jquery to find elements to find DOM elements. More details [here](https://github.com/san650/ember-cli-page-object/issues/289#issuecomment-289348460)

> native-event-context instead for context.js above?

yes, sure. Anyway I think we agreed on [`native-dom-context`](https://github.com/san650/ember-cli-page-object/pull/325#discussion_r143362645) name

Reply:
It seems that the core difference between these contexts is the way in which the events are triggered, so that's why I'm suggesting filenames with `event` in them as well as `native-event-context` instead of `native-dom-context`. Not a big deal either way...just pondering how to achieve the most clarity.

Reply:
Now I see your point about events. However `acceptance-jquery-event-context` is still not explanatory enough cause it also uses global ember test helpers(`find`, `click`). I'm also agree with you that it's not a big deal cause it's not a part of public API.
I think it might be better not too touch exiting context names at all until a better name eventually comes to us :)